app/main.py

```diff
@@ -1,7 +1,7 @@
 from app.knights import Knight
 
 
-def battle(knights):
+def battle(knights) -> dict:
```

Comment:
add type annotation to arg:)

Everything else looks OK, imo

Reply:
yes, sir!